src/screens/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { View, Text, ScrollView, KeyboardAvoidingView } from 'react-native'
+import { signIn } from 'src/store/actions'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { Heading, Input, Button, ButtonType, Link } from 'src/components'
+import { notification } from 'src/services'
+import { REGISTER_URL } from '../common'
+import { AppError, NavigationScreen } from 'src/common/enums'
+import { SignInValidationSchema } from './validationSchema'
+import styles from './styles'
+
+type RootStackParamList = {
+  [NavigationScreen.HOME]: undefined
+  [NavigationScreen.SIGN_IN]: undefined
+}
+
+type SignInScreenNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  NavigationScreen.SIGN_IN
+>
+
+type Props = {
+  navigation: SignInScreenNavigationProp
+}
+
+const SignIn: React.FC<Props> = ({ navigation }) => {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const dispatch = useDispatch()
+
+  const handleSignInSubmit = (): void => {
+    SignInValidationSchema.validate({ email, password })
+      .then(function (payload) {
+        dispatch(signIn(payload))
+        navigation.replace(NavigationScreen.HOME)
+      })
+      .catch(function (err) {
+        notification.error(
+          AppError.VALIDATION_ERROR,
+          JSON.stringify(err.message)
+        )
+      })
+  }
+
+  return (
+    <KeyboardAvoidingView
+      style={styles.keyBoardAvoidContainer}
+      behavior="height"
+    >
+      <ScrollView
+        contentContainerStyle={styles.scrollView}
+      >
+        <View>
+          <Heading style={styles.hello} label="Hello There" />
```

Comment:
```suggestion
          <Heading type={HeadingType.HUGE} label="Hello There" />
```
Set `/src/components/Heading/common/helpers` Huge style `fontSize: 36`